dynamic_programming/longest_increasing_subsequence.cpp

```diff
@@ -1,6 +1,9 @@
 // Program to calculate length of longest increasing subsequence in an array
-#include <bits/stdc++.h>
+#include <iostream>
+#include <limits.h>
```

Comment:
I guess this should be `climits` (as `limits.h` is in C language).
```suggestion
#include <climits>
```

Reply:
Made the suggested changes. Should work fine now 👍🏾